.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```

Comment:
Looks like this needs to be added to your `.gitignore`

Reply:
Added the .idea folder in full and removed this file from the remote.